### PR TITLE
feat(i18n): unify internalization texts usage

### PR DESCRIPTION
**Pull request by olafsulich**
## Description

**What's changed?**

- replaced double brackets `{{content}}` with single brackets `{content}`, this allows that to be compatible with react-intl (e.g. TM uses single brackets because it's using the `useIntl` hook instead of a custom function `t`)
  - so far, in our i18n files, we had both double and single brackets, depending on usage (either with react-intl or custom `t` function)
  - unifying this will help us to avoid errors like https://github.com/wireapp/wire-webapp/pull/18332, will reduce cognitive load, and thinking which way a dev should use
- removed `string.ts`, got rid of the `defineMessages`
- replaced `useIntl` hook usage with the custom `t` function, in this repo most of the internalization is handled by this function
- fixed tests, now all internalization strings are available, we don't have to check text by internalization id

**Potential further steps:**

- create a library to wrap internalization logic (we probably shouldn't use hooks from react-intl in one repo, and have a custom implementation in another one), it's not important now, though
- unify internalization between projects (via the library)

## Checklist

- [ ] mentions the JIRA issue in the PR name (Ex. [WPB-423])
- [x] PR has been self reviewed by the author;
- [ ] Hard-to-understand areas of the code have been commented;
- [ ] If it is a core feature, unit tests have been added;

[WPB-423]: https://wearezeta.atlassian.net/browse/WPB-423?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/script/util/ErrorUtil.ts [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+import {BackendErrorLabel, SyntheticErrorLabel} from '@wireapp/api-client/lib/http/';
+
+import {LabeledError} from '../auth/module/action/LabeledError';
+
+export const errorHandlerStrings = {
```
Comment (olafsulich) [inline src/script/util/ErrorUtil.ts:24]: Moved from `strings.ts`

File: src/script/util/ValidationUtil.ts [hunk 2]
```diff
@@ -93,3 +95,13 @@ export const assetV3 = (assetKey: string, assetToken?: string): true => {
   }
   return true;
 };
+
+export const validationErrorStrings = {
```
Comment (olafsulich) [inline src/script/util/ValidationUtil.ts:99]: Moved from `strings.ts`

File: src/script/auth/util/logoutUtil.ts [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+import {LOGOUT_REASON} from '../route';
+
+export const logoutReasonStrings = {
```
Comment (olafsulich) [inline src/script/auth/util/logoutUtil.ts:22]: Moved from `strings.ts`

File: src/script/auth/page/OAuthPermissions.tsx [hunk 4]
```diff
@@ -79,6 +79,13 @@ export enum Scope {
   READ_FEATURE_CONFIGS = 'read:feature_configs',
 }
 
+const scopeText: Record<Scope, string> = {
```
Comment (olafsulich) [inline src/script/auth/page/OAuthPermissions.tsx:82]: Moved from `strings.ts`

File: src/script/strings.ts [hunk 5]
```diff
@@ -1,885 +0,0 @@
-/*
```
Comment (olafsulich) [inline src/script/strings.ts:1]: an important change

File: src/script/util/LocalizerUtil/LocalizerUtil.ts [hunk 6]
```diff
@@ -120,8 +120,8 @@ export const LocalizerUtil = {
     };
 
     const substitutedEscaped = skipEscape
-      ? replaceSubstitute(value, /{{(.+?)}}/g, substitutions)
-      : replaceSubstituteEscaped(value, /{{(.+?)}}/g, substitutions);
+      ? replaceSubstitute(value, /{(.+?)}/g, substitutions)
+      : replaceSubstituteEscaped(value, /{(.+?)}/g, substitutions);
```
Comment (olafsulich) [inline src/script/util/LocalizerUtil/LocalizerUtil.ts:124]: an important change

File: src/script/util/LocalizerUtil/ReactLocalizerUtil.test.tsx [hunk 7]
```diff
@@ -119,14 +119,14 @@ describe('replaceReactComponents', () => {
   it('replaces components and literal strings at the same time', () => {
     const username1 = 'Tom';
     const username2 = 'Tim';
```
Comment (tlebon) [inline src/script/util/LocalizerUtil/ReactLocalizerUtil.test.tsx:121]: oh cool, i didn't realize i was immortalized in the code 👍🏻
thanks, Patryk.

**Review by przemvs — APPROVED** [review]

**Comment by tlebon** [comment]
theres a script in the package.json which references strings.ts and should be modified also.  L205
maybe good to check for other strings.ts references also.

**Review by aweiss-dev — APPROVED** [review]

**Review by e-maad — APPROVED** [review]
Great 👏

**Comment by olafsulich** [comment]
Thanks @tlebon, I've pushed changes (package.json, readme), don't see any other references

**Pull request merged**